feature/contributor/src/main/java/io/github/droidkaigi/confsched2020/contributor/ui/viewmodel/ContributorsViewModel.kt

```diff
@@ -33,28 +38,50 @@ class ContributorsViewModel @Inject constructor(
     private val contributorsLoadStateLiveData: LiveData<LoadState<List<Contributor>>> = liveData {
         emitSource(
             contributorRepository.contributorContents()
+                .dropWhileIndexed { index, value ->
+                    isEmptyCache(index, value)
+                }
                 .toLoadingState()
                 .asLiveData()
         )
 
         try {
             contributorRepository.refresh()
         } catch (exception: Exception) {
-            // We can show contributors with cache
+            reloadContributorsLiveData.value = LoadingState.Error(exception)
```

Comment:
It's a little difficult here, but I can't think of any better solution. 👀

Reply:
Yeah, I didn't like this, but I didn't have better ideas. Introducing another LiveData just for this error scenario would've been worse.